Fix #2175, Apply consistent Event ID names to common events

thnkslprpt opened the pull request:
**Checklist**
* [x] I reviewed the [Contributing Guide](https://github.com/nasa/osal/blob/main/CONTRIBUTING.md).
* [x] I signed and emailed the appropriate [Contributor License Agreement](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) to GSFC-SoftwareRelease@mail.nasa.gov and copied cfs-program@lists.nasa.gov.

**Describe the contribution**
- Fixes #2175
  - Consistent event IDs have been applied to the inconsistent cases to align them with a common Event ID naming convention.

**Testing performed**
Only GitHub CI actions.

**Expected behavior changes**
No impact on code behavior (no logic changes).
Consistent Event ID names for the events which are common to all/most cFS components and apps will improve consistency and ease make code review/debugging easier.

**Contributor Info**
Avi Weiss @thnkslprpt

thnkslprpt committed:
Fix #2175, Apply consistent naming to common Event IDs

jphickey reviewed (dismissed):
I think this is a good improvement on the event names, we should consider adding a paragraph to https://github.com/nasa/cFE/blob/main/docs/cFS_IdentifierNamingConvention.md that focuses on event names to document how the name should be constructed.

thnkslprpt commented:
> I think this is a good improvement on the event names, we should consider adding a paragraph to https://github.com/nasa/cFE/blob/main/docs/cFS_IdentifierNamingConvention.md that focuses on event names to document how the name should be constructed.

Yes good point - I will draft something up today and add it to that markdown doc.

thnkslprpt commented:
> I think this is a good improvement on the event names, we should consider adding a paragraph to https://github.com/nasa/cFE/blob/main/docs/cFS_IdentifierNamingConvention.md that focuses on event names to document how the name should be constructed.

I added the following section to the naming convention doc and squashed all the commits together now for merging.
![Screenshot 2022-11-01 08 52 00](https://user-images.githubusercontent.com/9024662/199125291-f95ac6f2-09b0-4beb-8d68-e02e7990df90.png)

thnkslprpt commented:
> I think this is a good improvement on the event names, we should consider adding a paragraph to https://github.com/nasa/cFE/blob/main/docs/cFS_IdentifierNamingConvention.md that focuses on event names to document how the name should be constructed.

@jphickey Joe - just making sure you're OK with the section I added to the naming convention doc.
Re-requested review as I added that commit here (and squashed it) after your original approval.
